pkg/storegateway/gateway_test.go

```diff
@@ -1068,6 +1092,7 @@ func mockGatewayConfig() Config {
 }
 
 func mockStorageConfig(t *testing.T) cortex_tsdb.BlocksStorageConfig {
+	//parallel testing causes data race
```

Comment:
```suggestion
```
Not needed

Reply:
@friedrichg I don't understand this one. Can you explain?

Reply:
mockStorageConfig is not a Test, is a mock helper function.
t.Parallel is only needed in Tests functions and t.Run statements. see https://github.com/golang/go/wiki/TableDrivenTests#parallel-testing